src/app/core/services/github.service.ts

```diff
@@ -246,6 +246,29 @@ export class GithubService {
     octokit.issues.updateLabel({ owner: ORG_NAME, repo: REPO, name: labelName, current_name: labelName, color: labelColor });
   }
 
+  /**
+   * Checks if the given list of users are allowed to be assigned to an issue.
+   * Throws an error if at least one of them is not allowed.
+   * @param assignees - GitHub usernames to be checked
+   */
+  areUsersAssignable(assignees: string[]): Observable<void> {
+    return from(
+      octokit.issues.listAssignees({
+        owner: ORG_NAME,
+        repo: REPO
+      })
+    ).pipe(
+      map((response: { data: { login: string }[] }) => response.data.map((user) => user.login)),
```

Comment:
The type annotation here is a little hard to read. I realize that octokit types are not installed. See if installing it allows type inference.
```suggestion
      map(({ data }: { data: { login: string }[] }) => data.map(({ login }) => login)),
```
You can also just use destructuring since you've typed it all out alr